src/kibana/plugins/visualize/editor/sidebar.html

```diff
@@ -1,7 +1,6 @@
 <div class="sidebar-container">
-  <form class="sidebar-list"
-        ng-class="{'hide-errors': hideErrors}"
-        ng-submit="visualizeEditor.$invalid ? dontApply() : stageEditableVis()"
+  <form class="sidebar-list in-time-errors"
+        ng-submit="stageEditableVis()"
```

Comment:
When a form is invalid the and you try to submit it, the browser doesn't allow the submit to go through, so that is a perfectly safe operation.